robottelo/ui/my_account.py

```diff
@@ -10,3 +11,48 @@ class MyAccount(Base):
     def navigate_to_entity(self):
         """Navigate to My Account page"""
         Navigator(self.browser).go_to_my_account()
+
+    def _update_helper(self, input_locator_name, new_value,
+                       value_locator_name=None):
+        """Update helper method for simple properties"""
+        input_locator = locators[input_locator_name]
+        value_locator = input_locator
+        if value_locator_name is not None:
+            value_locator = locators[value_locator_name]
+        self.navigate_to_entity()
+        self.assign_value(input_locator, new_value)
+        self.click(common_locators['submit'])
+        self.wait_until_element_is_not_visible(
```

Comment:
it will be nice to have a comment here, why we choose wait_until_element_is_not_visible vs  wait_until_element_exists, as the message overlap the account menu as not trivial at first look, what do you think?